controller/src/main/java/gov/va/vro/controller/VroController.java

```diff
@@ -76,6 +76,11 @@ public ResponseEntity<GeneratePdfResponse> generatePdf(GeneratePdfRequest reques
         request.getDiagnosticCode());
     try {
       GeneratePdfPayload model = generatePdfRequestMapper.toModel(request);
+      if (model.getPdfTemplate() == null
+          || model.getPdfTemplate().isEmpty()
+          || model.getPdfTemplate().isBlank()) {
```

Comment:
@anthonyvelazquez  There are ways to restrict enumeration in api spec in open api spec in general and java implementation of open api spec in particular. Not important enough for this pr but for future reference.